README.md

```diff
@@ -1,4 +1,139 @@
 
 # Holo Hosting Web SDK
+This module is designed so that web UIs can work directly with Holochain, or in the context of the
+Holo Hosting network.
+
 This package contains every component required to develop and test your web UI against your
 Holochain App Bundle.
+
+## Install
+
+```bash
+npm install @holo-host/web-sdk
+```
+
+## Setup
+
+### Starting your Holochain App Bundle
+A quick and simple script for running a hApp bundle is included in the web SDK.  This will setup and
+run a local Conductor configured to work with Chaperone's call specifications.
+
+```bash
+npx sdk-conductor --config conductor.json
+```
+
+Example `conductor.json`
+```javascript
+{
+    "key_store_file": "./happ-dev-keystore.key", // generated keystore file is saved here
+    "agent_id_file": "./happ-dev-agent-id.hcid", // generated keystore agent ID is saved here
+    "storage_dir": "./holochain-conductor/",     // instance memory is stored here
+    "log_level": null,
+    "sim2h_server": "http://localhost:9000",
+    "bundle": {
+        "id": "QmUgZ8e6xE1h9fH89CNqAXFQkkKyRh2Ag6jgTNC8wcoNYS",
```

Comment:
How do we get this `id`?

Initially I thought of it as an `id` that I get after registering the happ in the HHA, but since we are not using the HHA how do we get this `id`?

Reply:
`sdk-conductor` hasn't been developed yet, but the bundle ID can be anything during development.  It's just used to make your conductor instances unique.